### ISSUE TITLE
"Post-install page" appears every time Mozilla opens

### ISSUE THREAD

**Issue by Aydinv13** [AdguardTeam/BrowserAssistant#49]
https://uploads.adguard.com/Desktop_211124_0918.jpg

Ticket number: 555633

**Comment by ameshkov** [comment]
This is only possible if the browser storage is cleaned up on every startup

**Comment by ameshkov** [comment]
To figure that out we'll need user's configuration (the contents of `about:config`)

**Comment by Aydinv13** [comment]
@ameshkov yes, the browser's history is cleaned up when it closes, but it looks like a bug because nothing helps to use this option when Assistant is on.

**Comment by ameshkov** [comment]
If Firefox considers extension's `localStorage` a part of the browsing history, this is a problem.

We'll have to start using different type of storage to store the user's consent, and this requires adding more permissions.

**Comment by maximtop** [comment]
According to the firefox documentation adding the storage permission won't trigger alert about new permissions https://extensionworkshop.com/documentation/develop/request-the-right-permissions/#advised-permissions
